src/Resources/public/js/autocomplete-create.js

```diff
@@ -0,0 +1,127 @@
+                },
+                // to indicate that infinite scrolling can be used
+                processResults: function (data, params) {
+                    return {
+                        results: data.results,
+                        pagination: {
+                            more: data.has_next_page
+                        }
+                    };
+                },
+                cache: true
+            },
+            placeholder: '',
+            allowClear: true,
+            minimumInputLength: 1,
+            language: {
+                noResults: function () {
+                    return noResultsCreate(url_action, select_id, field_name);
+                }
+            },
+            escapeMarkup: function (markup) {
+                return markup;
+            }
+        });
+    });
+}
+
+function ajaxModalEntityAction(url_action, select_id, field_name) {
+    $('#'+select_id).select2('close');
+    $.ajax({
+        url : url_action,
+        type: 'GET',
+        success: function(data) {
+            showModalEntityForm(data, url_action, field_name, select_id);
+            $('#modal-entity-form').modal({ backdrop: true, keyboard: true });
+        }
+    });
+}
+function showModalEntityForm(data, url_action, field_name, select_id) {
+    var content = $('#modal-entity-form p.modal-body-content');
+    content.html(data.template);
+    $('form[name="'+field_name+'"]').attr('action', url_action);
+    initAjaxForm(field_name, select_id);
+}
+function initAjaxForm(field_name, select_id) {
+    $('form[name="'+field_name+'"]').submit(function( event ) {
+        event.preventDefault();
+        var url_action = $(this).attr('action');
+        $.ajax({
+            url: url_action,
+            type: $(this).attr('method'),
+            data: $(this).serializefiles(),
+            cache: false,
+            contentType: false,
+            processData: false,
+            success: function(data) {
+                if (data.hasOwnProperty('option')) {
+                    $('#modal-entity-form').modal('hide');
+                    var newOption = new Option(data.option.text, data.option.id, true, true);
+                    $('#'+select_id).append(newOption).trigger('change');
+                    // manually trigger the `select2:select` event
+                    $('#'+select_id).trigger({
+                        type: 'select2:select',
+                        params: {
+                            data: data.option
+                        }
+                    });
+                }
+                if (data.hasOwnProperty('template')) {
+                    showModalEntityForm(data, url_action, field_name, select_id);
+                }
+            },
+            error: function(error){
+                console.log(error);
+            }
+        });
+    });
+}
+//USAGE: $("#form").serializefiles();
+(function($) {
+$.fn.serializefiles = function() {
+    CKupdate();
```

Comment:
Where does CKeditor act in the feature ?

Reply:
It is only necessary if you use it. It could be removed

Reply:
This is the limit of such feature : loading JS widgets requires to initialize them. But it's not this bundle responsibility to handle that. You had the case for CKEditor, but you could encounter it for color picker, calendar or any JS widget ;-)